mach

```diff
@@ -42,30 +43,42 @@ from tools.utils import (
 # === SUBCOMMANDS === #
 
 
-def _install(prefix=None, config=None):
-    configuration = "Debug"
-    if config:
-        configuration = config
-    cmake_cmd = ["cmake", "--install", "build", "--config", configuration]
+def _install(prefix=None, config="Release"):
+    cmake_cmd = ["cmake", "--install", "build", "--config", config]
     if prefix:
         cmake_cmd.extend(["--prefix", prefix])
     print(cmake_cmd)
-    subprocess.run(cmake_cmd, check=True)
+    if subprocess.call(cmake_cmd) == 1:
+        print("`cmake --install` returned with an error.")
+        if config == "Debug":
+            print(
+                "Please make sure to run `./mach build` before `./mach install --debug`."
+            )
+        else:
+            print(
+                "Please make sure to run `./mach build --release` before `./mach install`."
+            )
 
 
 @subcommand(
     [
         argument("-p", "--prefix", help="The path prefix to install into.", type=str),
-        argument(
-            "-c",
-            "--config",
-            help="The config to install, i.e. Debug or Release.",
-            type=str,
-        ),
+        argument("--debug", help="Install a debug build.", action="store_true"),
     ]
 )
 def install(args):
-    _install(prefix=args.prefix, config=args.config)
+    if args.debug and args.release:
```

Comment:
This will crash now 🙂 
You can drop the entire condition.

Reply:
Oops :-) Fixed.